src/poprox_storage/repositories/accounts.py

```diff
@@ -55,6 +55,21 @@ def fetch_account_by_email(self, email: str) -> Account | None:
             return accounts[0]
         return None
 
+    def store_account(self, account: Account) -> UUID | None:
```

Comment:
To match existing semantics, should this insert or update?

Reply:
It should probably upsert